src/utils/tekton-results.ts

```diff
@@ -303,6 +303,14 @@ const getFilteredPipelineRuns = (
     nextPageToken,
     cacheKey,
   );
+  // When pipelines are running, the etcd would keep their results.
+  // deleting pipelines frome ectd would make unknown tekton results.
+  // Just to get meaningful test runs, we need to filter 'unknown' out.
+  const filteredPipelineRuns = originalPipelineRuns.filter((pipelinerun) => {
+    return pipelinerun.status?.conditions?.every((c) => c.status !== 'Unknown') ?? true;
+  });
+  return [filteredPipelineRuns, list];
+};
```

Comment:
Is it possible that an unknown status will appear for the pipeline runs that were not deleted? In that case, It's not a good idea to filter out the `Unknown` status pipeline runs.

Reply:
Can you specify why it is not good idea?

Reply:
I just improved the filter condition to ensure it is more accurate and added more explanations. Could you help to review again? Thank you!